pkg/utils/pretty/pretty.go

```diff
@@ -77,3 +78,7 @@ func Map[K constraints.Ordered, V any](values map[K]V, maxItems int) string {
 	}
 	return buf.String()
 }
+
+func Taint(t v1.Taint) string {
+	return fmt.Sprintf("%s=%s:%s", t.Key, t.Value, t.Effect)
```

Comment:
If there's no value, this is an empty =, can you adjust this so we don't include the = if we don't need to?